### PR TITLE
Rename and Update WyndhamHotels.com.xml, Fix #12672

### LINKED ISSUE #12672: Spliting Wyndham.xml into Separate Rulesets
Type: other

The goal is to remove the wildcard `targets` and simplify this ruleset,

Part of #11203, [`Wyndham.xml`](https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/Wyndham.xml), [`Wyndham-mismatches.xml`](https://github.com/EFForg/https-everywhere/blob/master/src/chrome/content/rules/Wyndham-mismatches.xml)

- [x] CanvasHolidays.co.uk (#12655)
- [x] Cheznous.com (#12673) 
- [x] CompetitionsByWyndham.com.au #13974 
- [x] Cottages.com (#13067)
- [x] Cottageselection.co.uk #13849 [REMOVED]
- [x] Cottagesdirect.co.uk (#14351)
- [x] English-Country-Cottages.co.uk (#13837)
- [x] EasyCottages.com (#12678) [REMOVED]
- [x] French-Country-Cottages.co.uk (#13758)
- [x] Hoseasons.co.uk (#12657)  
- [x] Individual-villas.co.uk (#13958) [REMOVED]
- [x] Irish-Country-Cottages.co.uk (#13371)
- [x] Italian-Country-Cottages.co.uk (#13837)
- [x] JamesVillas.co.uk (#12795)
- [x] Ovscruise.com (#13373)
- [x] RCI.com (#14507)
- [x] RCITravelStore.co.uk (#12656)
- [x] Resortquestsecure.com (#13850) [REMOVED]
- [x] Secureholidays.com (#14059) [REMOVED]
- [x] WelcomeCottages.com (#13996)
- [x] WyndhamJobs.com (#13382)
- [x] WyndhamVacationRentals.com (#14459)
- [x] Wyndhamvrap.com (#13375)
- [x] Landal.* (#13851)
- [x] Landalparkshop.de (#13838) 
- [x] Landalskilife.* (#13570)
- [x] LandalCampings.de (#13759)
- [x] Mijnlandal.nl (#13957) [REMOVED]
- [x] _Wyndham-mismatches.xml_ (#14064) [REMOVED]

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

File: src/chrome/content/rules/Hoseasons.co.uk.xml [hunk 1]
```diff
@@ -32,15 +34,13 @@
 <ruleset name="Hoseasons.co.uk">
 	<target host="hoseasons.co.uk" />
 	<target host="www.hoseasons.co.uk" />
-	<target host="api.hoseasons.co.uk" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Hoseasons.co.uk.xml]: Works for me

File: src/chrome/content/rules/Hoseasons.co.uk.xml [hunk 2]
```diff
@@ -32,15 +34,13 @@
...
 	<target host="autodiscover.hoseasons.co.uk" />
 	<target host="booking.hoseasons.co.uk" />
 	<target host="goschedulev2.hoseasons.co.uk" />
 	<target host="images.hoseasons.co.uk" />
 	<target host="partners.hoseasons.co.uk" />
 	<target host="procurement.hoseasons.co.uk" />
 	<target host="www.procurement.hoseasons.co.uk" />
-	<target host="productfeed.hoseasons.co.uk" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Hoseasons.co.uk.xml]: Works for me

File: src/chrome/content/rules/Ovscruise.com.xml [hunk 3]
```diff
@@ -1,7 +1,10 @@
 <!--
-	For other Wyndham related rulesets, see Wyndham.xml
+	For other Wyndham related rulesets, see WyndhamHotels.com.xml
 
 	Non-functional hosts
+		Connection refused:
+		- prod-cruise.ovscruise.com
```
Comment (Bisaloo) [inline src/chrome/content/rules/Ovscruise.com.xml]: This subdomain is unreachable over HTTP as well. We can completely remove it.

File: utils/ruleset-whitelist.csv [hunk 4]
```diff
@@ -1,4 +1,5 @@
 hash,auto-pass coverage test,auto-pass fetch test,filename
+030dda050df8728653c6965bc82a9efbb05f80f04e3f3379a7ef3e114fb94e33,0,1,WyndhamHotels.com.xml
```
Comment (Bisaloo) [inline utils/ruleset-whitelist.csv]: Can we just retry without whitelisting this ruleset? It may be a transient failure.

**Comment by cschanaj** [comment]
@Bisaloo travis is still failing without the whitelisting `WyndhamHotels.com.xml`. Maybe we should whitelist this ruleset?

**Comment by Bisaloo** [comment]
Okay, if fails with `curl` as well. Let's whitelist it.

**Comment by Bisaloo** [comment]
Sorry, you were right: `api.hoseasons.co.uk` and `productfeed.hoseasons.co.uk` are misconfigured. They are serving both the correct and expired cert and should therefore be removed.

https://www.ssllabs.com/ssltest/analyze.html?d=api.hoseasons.co.uk

**Comment by cschanaj** [comment]
@Bisaloo I agree, Travis pass and fail inconsistently. Removed the domains from the rulesets.

**Pull request merged**